feat(content): New human missions and jobs (#8207)
endless-sky/endless-sky: data/human/human missions.txt

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Karen Transport 1"
	name "Passenger transport to <planet>"

> 	description "Bring <passengers> to <destination>."

Reviewer: ```suggestion
	description "Bring <fare> to <destination> by <date>."
```

	passengers 1
	deadline 2 0
	source
		government "Republic"
		attributes "paradise"
	destination
		distance 1 1
		attributes "paradise"
	to offer
		random < 1
		"flagship crew" > 3
	on offer
		conversation
			`As you are wandering the spaceport, a woman with extravagant clothing approaches you. "I am to be transported to <destination> immediately. You will be paid well."`
			choice
				`	(Invite her into your ship.)`
					goto accept
				`	"I'd expect my passengers to be more respectful."`
			`	"Well, you just lost yourself a passenger!" She storms off angrily, cursing the audacity of "these space freaks."`
				decline
			
			label accept

> 			`	She takes off towards your ship without waiting for you. When you catch up to her, she is standing in a passageway, shouting at your crew to take her bags away. You tell her that your crew is busy and passengers are expected to carry their own luggage then leave for your cabin.`

Author: @EjoThims removed that comma, I'll let the two of you fight it out.

> 			`	She takes off towards your ship without waiting for you. When you catch up to her, she is standing in a passageway, shouting at your crew to take her bags away. You tell her that your crew is busy and passengers are expected to carry their own luggage then leave for your cabin.`

Reviewer 2: I wouldn't fight anyone over it, but, to me, it came down to that `You tell her that your crew is busy then leave for your cabin.` wouldn't warrant a comma, either.

				accept
	on enter
		dialog
			`During takeoff, you keep hearing loud banging and cursing from one of the passenger cabins. Later on, you hear the woman you just picked up screeching at your crew for their 'gross incompetence' and 'boneheaded flying.'`
			`	You can barely restrain yourself from making a snappy remark, but you have learned that it's best not to antagonize your passengers.`
	on enter
		conversation

> 			`You hear hurling and splashing noises in the background. It seems your passenger threw up, not uncommon for first-timers.`

Reviewer: One caveat here is that a player can trigger this dialog by just taking off and landing again. Probably not worth dropping it for that edge case, but in case we ever get something like, `on enter destination`, that would be worth swapping it over.

				`	You make sure to lock the cockpit door this time and then carefully stop the ship using the thrusters.`
	on enter
		conversation
			`You glance behind you to make sure the cockpit doors are still closed. Maybe this time you can maneuver the ship in peace.`
			`	Just as you finished that thought, the intercom beeps. "Are you sure you can handle landing by yourself? Don't be afraid to ask for help, it isn't embarrassing. Pilot training is worthless anyway."`
			choice
				`	"Do you even know how to fly a ship?"`
					goto fly
				`	(Stay silent.)`
					goto silent

			label fly
			`	"I never bothered to apply for a licene, but I know my way around the controls," he says. You remain dubious, considering he didn't even know how to stop a ship.`
			choice
				`	"If you are not a pilot, then stop bothering me."`
					goto stop
				`	(Accept his advice.)`
					goto silent

			label silent
			`	You silently listen to the 'advice' he is giving. "Slow down, you are going way too fast", "you are turning the wrong way", "be more gentle on those controls". When he gets to "Why are we not there yet?" you decide to mute the audio, and focus on flying the ship.`
				decline

			label stop

> 			`	"This is what I get for helping you? Very grateful of you. Maybe next time you won't get a passenger that flies the ship for you, and you will crash into a planet."`

Reviewer 3: ```suggestion
			`	"This is what I get for helping you? Very grateful of you. Maybe next time you won't get a passenger that flies the ship for you, and you'll crash into a planet!"`
```

			`	You didn't think silencing the audio could be this satisfying. Maybe now you can finally focus on flying the ship.`
	on enter
		conversation

> 			`"You should have let me take the wheel, captain," your middle-aged passenger says when you tell him that you won't get to <destination> in time."`

Reviewer: ```suggestion
			`"You should have let me take the wheel, captain," your middle-aged passenger says when you tell him that you won't get to <planet> in time."`
```

> 			`"You should have let me take the wheel, captain", your middle-aged passenger says when you tell him that you won't get to <destination> in time."`

Reviewer 3: ```suggestion
			`"You should have let me take the wheel, captain," your middle-aged passenger says when you tell him that you won't get to <destination> in time."`
```

			choice
				`	"Fine. Do it yourself."`
					goto pilot
				`	"I'm rather fond of living, actually."`
			`	"You think I would have crashed your ship? Nonsense! I can fly this rustbucket better than anyone!" he says.`
			`	You watch as he walks towards the cockpit.`
			choice
				`	(Stop him.)`
				`	(Let him fly the ship.)`
					goto pilot
			`	You quickly catch up to him before politely telling him to go back to his compartments and stay there for the rest of the journey.`
				decline
			
			label pilot
			`	He straps himself in, and begins fiddling with the controls. Navigation lights. Life support. Thrusters.`

> 			`	The ship suddenly lurches to the side, and you find yourself on the ground. "No need to panic, everything is under control", the bogus pilot exclaims.`

Reviewer 3: ```suggestion
			`	The ship suddenly lurches to the side, and you find yourself on the ground. "No need to panic, everything is under control," the bogus pilot exclaims.`
```

	passengers 5
	source
		attributes "dirt belt"
	destination
		distance 4
		attributes "dirt belt"
	to offer
		random < 1
		"flagship crew" > 5
	on offer
		conversation
			`As you are wandering the spaceport, a group of teenage boys huddling in a corner catches your eye. They seem to be talking amongst themselves, sometimes throwing a glance towards you.`
			choice
				`	(Approach them.)`
					goto ask
				`	(Walk away.)`
			`	Just as you turn away to leave, one of the teenagers straightens up and approaches you.`
				goto introduction

			label ask
			`	"Can I help you?"`
			`	One of the teens steps forward and introduces himself.`

			label introduction
			`	"Hello, I'm Howard. Me and my four friends are looking for transport off-world, to <destination>, but cannot afford to pay for it. Can you help us?"`
			choice
				`	"Sure."`
					goto accept
				`	"Why do you want to go there?"`
				`	"Sorry, I'm not headed that way."`
					goto decline
			`	After a moment of silence, Howard replies. "We heard they have well-paying jobs over there, even for teenagers. And we don't want to work for the Syndicate or those minted folks on the paradise worlds."`

> 			`	He pauses for a second, then continues, "We grew up in the dirt belt. This is our home. We don't want to leave all that behind." You hear something strange in his voice, but cannot quite make sense of it. It might just be their desperation.`

Reviewer 3: ```suggestion
			`	He pauses for a second, then continues, "We grew up in the Dirt Belt. This is our home, we don't want to leave all that behind." You hear something strange in his voice, but cannot quite make sense of it. It might just be their desperation.`
```

Author: ```suggestion
			`	He pauses for a second, then continues, "We grew up in the Dirt Belt. This is our home. We don't want to leave all that behind." You hear something strange in his voice, but cannot quite make sense of it. It might just be their desperation.`
```

			`	You freeze. Larry has your own sidearm in his hand, pointed at you. He must have taken it last night, when you thought they were partying. Instantly, all drunkenness is gone from him and his friends, and they eye you with sudden glee and hostility. "That's right," Howard says, with a snark. "Now, will you please let us into the cockpit?"`
			`	You step away from the door, careful not to make any sudden moves.`
				goto options
			
			label come
			`	You let the group into the cockpit. Four of them immediately crowd the place, but the fifth teen - the one leaning on the wall - keeps his distance.`
			`	The group seems intrigued by the various controls and equipment that are all over the cockpit. They ask you questions and walk around with clear curiosity. Then, all of a sudden, those inside grab you and try to pin your hands down.`
			choice
				`	(Call for help.)`
				`	(Fight them.)`
					goto fight
				`	(Reach for my sidearm.)`
					goto sidearm
			`	You try to shout, but a kidney punch knocks the wind out of you before you can speak.`
				goto fight

			label sidearm
			`	You try to reach for your pistol, but it's no longer in its holster.`

			label fight

> 			`	You stand no chance against the four attackers. They each grab one of your limbs and immobilize you, then the fifth guy points your own gun at you. They must have taken it last night, when you thought they were partying.`

Reviewer 4: ```suggestion
			`	You stand no chance against the four attackers. They each grab one of your limbs and immobilize you as the remaining boy points your own gun at you. They must have taken it last night, when you thought they were partying.`
```

			`	"Keep it steady, captain," Howard says with glee from your right side. "No funny business. Larry here has an itchy trigger finger." He then lets go of your right leg he has been holding onto, and addresses Larry. "Told you it would work! Next time, I get the gun while you fight in the dirt with the idiot." You notice how none of them seem to be drunk in the slightest anymore.`

			label options
			`	"You will fly the ship for us. If you don't try anything, we'll let you go afterwards." You don't give much credit to their promise.`
			choice
				`	(Pretend to comply with the demands.)`
					goto comply
				`	(Shout for your crew.)`

			branch survived
				random < "flagship crew" - 1
			`	Larry fires the weapon, and you feel sickening pain in your chest. You fall to the ground and try to gasp for air, but it feels too hard. You watch as everything goes black and your consciousness slips away.`
				die

			label survived
			`	Larry fires the weapon, and you feel sickening pain in your left shoulder. Before Larry can fire again however, a member of your crew pops up behind him and takes him out with a well-aimed shot.`

> 			`	With the other four attackers being unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and stitch up your wound. You still can't quite believe what just happened.`

Reviewer 4: ```suggestion
			`	With the other four attackers left unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and stitch up your wound. You still can't quite believe what just happened.`
```

			`	You thank your fellow crew members for saving your life and decide to not let your sidearm out of your sight ever again.`
				decline

			label comply
			`	You slowly sit down in the pilot chair and carefully guide the ship towards the spot Howard indicates on the map. At an opportune moment, you ram the thrusters up to maximum and turn the ship to the side. Howard and his friends fall over, but you remain secure in your chair.`
			`	You run over to Larry and make a quick grab for the weapon before he can recover. Now that you have the upper hand, you feel confident to call for your crew.`
			`	As they take away the five hijackers, the magnitude of the events slowly sinks in. You could easily have died today.`
			`	You thank your fellow crew members and decide not to let your sidearm out of your sight ever again.`
	on complete
		payment 50000
		dialog
			`You notify the spaceport authorities about the situation and hand over the teenagers to the security detachment they send out.`

> 			`	"That's the 'Howard Gang'! They had been causing trouble on <origin>, then skipped town when they caught wind of the investigation. We are glad you managed to catch them."`

Reviewer 4: ```suggestion
			`	"That's the 'Howard Gang'! They had been causing trouble on <origin>, then skipped town when they caught wind of the investigation. We're glad you managed to catch them."`
```

> 			`	They give you <payment> bounty for apprehending the criminals.`

Reviewer 3: ```suggestion
			`	They give you a bounty of <payment> for apprehending the criminals.`
```

			`	"Idiot! You flying space morons can't even read a map! We are nowhere near <planet>! Are you trying to kidnap me? I'll get the Navy to put you into prison for this!" You try to speak, but can't get a word in edgeways.`

			label restrain
			`	You call your crew, and together you manage to detain her. She is now locked up in her cabin, but her screams still echo in your head. This will be a long day.`
			`	You decide to drop her off on the first planet you land on.`
	on complete
		conversation
			`You are glad to finally see your passenger leaving, but then you realize she has't paid you yet. You decide to confront her about it.`
			`	"You expect ME to pay after all that? You are lucky I'm not suing you for damages!" She stops for a moment, then throws a credit chip at you. "Fine, take this, but don't bother me again!" She stomps off angrily.`
			`	You look at the credit chip she had so generously given you. It's worth... 10 credits. Oh well.`
		payment 10
	on abort
		conversation
			branch landed
				has "flagship landed"
			`You decide you had enough of her constant fits, and inform her that her journey is over.`

> 			`	As you listen with grim satisfaction to the air hissing out of the opening airlock, you take comfort in the knowledge that you at least got the bunk space back.`

Author: This is the unspoken truth about ES. When you abort a mission mid-flight, you get the cargo and bunk space back - and there isn't a lot you can do with those people in the middle of deep space.

> 			`	As you listen with grim satisfaction to the air hissing out of the opening airlock, you take comfort in the knowledge that you at least got the bunk space back.`

Reviewer 2: That's true, yes. I just meant I'm not so this mission on it's own is the place to officially canonize/denote that as literal within the abstraction of the game. Like, if we're going to recognize that as a literal occurrence, there should be some kind of specific consequences to address that for spacing Katya or Ijs (for example) beyond -just- failing the campaign, and we may even want to address consequences for repeatedly doing so on this and other random jobs.

If no one else has a problem with it being introduced here specifically, I don't consider it any kind of block, just musing about whether or not adding it back here alongside also accounting for it elsewhere may be better as a separate thing with more focused scrutiny.

> 			`	As you listen with grim satisfaction to the air hissing out of the opening airlock, you take comfort in the knowledge that you at least got the bunk space back.`

Reviewer 5: [From Pointed](https://github.com/endless-sky/endless-sky/pull/5781#discussion_r593755478):

>Think about how this would work in real life: if you directly and negligently facilitated the death of 5-10 innocent yet stupid people what's gonna happen? A 10k fine? Or imprisonment for a decade or more?
>
>I know games aren't real life, but you can't bend the rules of physics or how human societies work too much, or else it breaks the player out of the illusion that the game is crafting.

Also, just because something happens in gameplay doesn't mean that it is codified in story. See Derpy's comments on the no-mercy Navy in Hai Revealed.

> 			`	As you listen with grim satisfaction to the air hissing out of the opening airlock, you take comfort in the knowledge that you at least got the bunk space back.`

Reviewer 4: I would like to have booting people out midflight be acknowledged in some way, but there definitely should be actual consequences if we're going to do that.

			`	After a moment of relative calm, the world begins to spin around you with sickening speed. You hear alarms go off all over the ship, and sparks fly all around you. It seems that he managed to put the ship into a spin by overloading some of the engines.`
			`	You try to get up, but it is impossible to find your balance in the wheeling ship. The alarms get more frantic with every passing second, until you hear a low rumbling, then a thundering explosion consumes everything.`
				explode
	on complete
		conversation
			`After landing, you open the doors to let your passengers out. You already see the middle-aged man approaching you.`
			`	"Took you long enough! Anyway, I must get going now. Some people can't sit around all day."`
			choice
				`	(Ask him for payment.)`
				`	(Let him go.)`
					decline
			`	"Pay you?! You would have crashed the ship, had I not helped you! I've practically flown your ship here, and you have the guts to ask for payment?"`
			`	He turns his back to you and walks off. You consider calling spaceport security, but decide against it; the sooner you get rid of him, the better.`

> 	on abort

Author: I needed this when this mission used `on fail`, but I ended up changing that later, so this isn't needed anymore.

			label options
			`	"You will fly the ship for us. If you don't try anything, we'll let you go afterwards." You don't give much credit to their promise.`
			choice
				`	(Pretend to comply with the demands.)`
					goto comply
				`	(Shout for your crew.)`

			branch survived
				random < "flagship crew" - 1
			`	Larry fires the weapon, and you feel sickening pain in your chest. You fall to the ground, and try to gasp for air, but it feels too hard. You watch as everything goes black and your consciousness slips away.`
				die

			label survived
			`	Larry fires the weapon, and you feel sickening pain in your left shoulder. Before Larry could fire again however, a member of your crew pops up behind him and takes him out with a well-aimed shot.`
			`	With the other four attackers being unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and stitch up your wound. You still can't quite believe what just happened.`

> 			`	You thank your fellow crew members for saving your life, and decide to not let your sidearm out of your sight ever again.`

Author: I don't think there is anything unreasonable about paying more attention to your weapon, especially after nearly getting shot by that very same weapon.

				decline

			label comply
			`	You slowly sit down in the pilot chair, and carefully guide the ship towards the spot Howard indicated on the map. At an opportune moment, you ram the thrusters up to maximum, and turn the ship to the side. Howard and his friends fall over, but you remain secure in your chair.`
			`	You run over to Larry and make a quick grab for the weapon before he can recover. Now that you have the upper hand, you feel confident to call for your crew.`
			`	As they take away the five hijackers, the magnitude of the events slowly sinks in. You could easily have died today.`
			`	You thank your fellow crew members, and decide not to let your sidearm out of your sight ever again.`
	on complete
		payment 50000
		dialog
			`You notify the spaceport authorities about the situation, and hand over the teenagers to the security detachment they sent out.`

> 			`	"That's the 'Howard Gang'! They have been causing trouble on <origin>, then turned tail when they caught wind of the investigation. We are glad you managed to catch them."`

Reviewer 2: ```suggestion
			`	"That's the 'Howard Gang'! They had been causing trouble on <origin>, then skipped town when they caught wind of the investigation. We are glad you managed to catch them."`
```
Probably a better phrase for the intent here? Tense swap is the other way around this time though

	on enter
		conversation
			`"You should have let me take the wheel, captain," your middle-aged passenger says when you tell him that you won't get to <planet> in time."`
			choice
				`	"Fine. Do it yourself."`
					goto pilot
				`	"I'm rather fond of living, actually."`
			`	"You think I would have crashed your ship? Nonsense! I can fly this rustbucket better than anyone!" he says.`
			`	You watch as he walks towards the cockpit.`
			choice
				`	(Stop him.)`
				`	(Let him fly the ship.)`
					goto pilot
			`	You quickly catch up to him before politely telling him to go back to his compartments and stay there for the rest of the journey.`
				decline
			
			label pilot
			`	He straps himself in, and begins fiddling with the controls. Navigation lights. Life support. Thrusters.`
			`	The ship suddenly lurches to the side, and you find yourself on the ground. "No need to panic, everything is under control," he exclaims.`
			`	After a moment of relative calm, the world begins to spin around you with sickening speed. You hear alarms go off all over the ship, and sparks fly all around you. It seems that he managed to put the ship into a spin by overloading some of the engines.`

> 			`	You try to get up, but it is impossible to find your balance in the wheeling ship. The alarms get more frantic with every passing second, until you hear a low rumbling, then a thundering explosion consumes everything.`

Reviewer 6: That still seems a little off. Maybe:

```suggestion
			`	You try to get up, but it is impossible to find your balance in the wheeling ship. The alarms get more frantic with every passing second, until you hear a low rumbling, until a thundering explosion consumes everything.`
```

or

```suggestion
			`	You try to get up, but it is impossible to find your balance in the wheeling ship. The alarms get more frantic with every passing second, until you hear a low rumbling, followed by a thundering explosion which consumes everything.`
```

				`	(Reach for my sidearm.)`
					goto sidearm
			`	You try to shout, but a kidney punch knocks the wind out of you before you can speak.`
				goto fight

			label sidearm
			`	You try to reach for your pistol, but it's no longer in its holster.`

			label fight
			`	You stand no chance against the four attackers. They each grab one of your limbs and immobilize you as the remaining boy points your own gun at you. They must have taken it last night, when you thought they were partying.`
			`	"Keep it steady, captain," Howard says with glee from your right side. "No funny business. Larry here has an itchy trigger finger." He then lets go of your right leg he has been holding onto, and addresses Larry. "Told you it would work! Next time, I get the gun while you fight in the dirt with the idiot." You notice how none of them seem to be drunk in the slightest anymore.`

			label options
			`	"You will fly the ship for us. If you don't try anything, we'll let you go afterwards." You don't give much credit to their promise.`
			choice
				`	(Pretend to comply with the demands.)`
					goto comply
				`	(Shout for your crew.)`

			branch survived
				random < "flagship crew" - 1

> 			`	Larry fires the weapon, and you feel a sickening pain in your chest. You fall to the ground and try to gasp for air, but it feels too hard. You watch as everything goes black and your consciousness slips away.`

Reviewer 5: ```suggestion
			`	As you shout, Larry fires, and you feel a sickening pain in your chest. You fall to the ground and try to gasp for air, but it feels too hard. You watch as everything goes black and your consciousness slips away.`
```

				die

			label survived

> 			`	Larry fires the weapon, and you feel sickening pain in your left shoulder. Before Larry can fire again however, a member of your crew pops up behind him and takes him out with a well-aimed shot.`

Reviewer 5: ```suggestion
			`	As you shout, Larry fires the weapon, and you feel a sickening pain in your left shoulder. Before Larry can fire again, however, a member of your crew pops up behind him and takes him out with a well-aimed shot.`
```

> 			`	With the other four attackers left unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and stitch up your wound. You still can't quite believe what just happened.`

Reviewer 5: ```suggestion
			`	With the other four attackers left unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and treat your wound.`
```

> 			`	You thank your fellow crew members for saving your life and decide to not let your sidearm out of your sight ever again.`

Reviewer 5: ```suggestion
			`	You thank your fellow crew members for saving your life and vow to never let your sidearm out of your sight ever again.`
```

Reviewer 6: ```suggestion
			`	You thank your fellow crew members for saving your life and vow never to let your sidearm out of your sight ever again.`
```

This seems a little more fluid, although it would be fine either way.

				decline

			label comply

> 			`	You slowly sit down in the pilot chair and carefully guide the ship towards the spot Howard indicates on the map. At an opportune moment, you ram the thrusters up to maximum and turn the ship to the side. Howard and his friends fall over, but you remain secure in your chair.`

Reviewer 5: If we're close enough to the planet to follow a map, shouldn't this be part of the `on complete`?

> 			`	You run over to Larry and make a quick grab for the weapon before he can recover. Now that you have the upper hand, you feel confident to call for your crew.`
> 			`	As they take away the five hijackers, the magnitude of the events slowly sinks in. You could easily have died today.`
> 			`	You thank your fellow crew members and decide not to let your sidearm out of your sight ever again.`

Reviewer 5: ```suggestion
			`	You run over to Larry and make a quick grab for the weapon before he can recover. Now that you have the upper hand, you feel confident to call for your crew. You thank your fellow crew members and vow to never let your sidearm out of your sight ever again.`
```

	on complete
		payment 50000
		dialog
			`You notify the spaceport authorities about the situation and hand over the teenagers to the security detachment they send out.`

> 			`	"That's the 'Howard Gang'! They had been causing trouble on <origin>, then skipped town when they caught wind of the investigation. We're glad you managed to catch them."`

Reviewer 5: ```suggestion
			`	"That's the 'Howard Gang'!" one of the officers says. "They had been causing trouble on <origin>, then skipped town when they caught wind of the investigation. We're glad you managed to catch them."`
```

				decline

			label stop
			`	"This is what I get for helping you? Very grateful of you. Maybe next time you won't get a passenger that flies the ship for you, and you'll crash into a planet!"`
			`	You didn't think silencing the audio could be this satisfying. Maybe now you can finally focus on flying the ship.`
	on enter
		conversation
			`"You should have let me take the wheel, captain," your middle-aged passenger says when you tell him that you won't get to <planet> in time."`
			choice
				`	"Fine. Do it yourself."`
					goto pilot
				`	"I'm rather fond of living, actually."`
			`	"You think I would have crashed your ship? Nonsense! I can fly this rustbucket better than anyone!" he says.`
			`	You watch as he walks towards the cockpit.`
			choice
				`	(Stop him.)`
				`	(Let him fly the ship.)`
					goto pilot
			`	You quickly catch up to him before politely telling him to go back to his compartments and stay there for the rest of the journey.`
				decline
			
			label pilot
			`	He straps himself in, and begins fiddling with the controls. Navigation lights. Life support. Thrusters.`

> 			`	The ship suddenly lurches to the side, and you find yourself on the ground. "No need to panic, everything is under control," the bogus pilot exclaims.`

Reviewer 4: ```suggestion
			`	The ship suddenly lurches to the side, and you find yourself on the ground. "No need to panic, everything is under control," he exclaims.`
```

			choice
				`	(Investigate the noise.)`
					goto investigate
				`	(Don't investigate.)`
			`You continue on your way to your quarters without noticing anything strange. The noise must have been some piece of equipment you haven't noticed before.`
				decline
			label investigate
			`	You quietly walk closer, then stop and listen to what seems to be people talking.`
			`	"...seems so nice. Picked us up and all."`
			`	"Nah. They all expect us to pay them later, these guys. They work for money, like everyone else. We're just an investment."`
			`	"We can't back out now. There's no way we could get there otherwise. Don't act like it's your first time in the business." In the third speaker, you recognize Howard, one of your teenage passengers.`
			choice
				`	(Confront them.)`
				`	(Walk away.)`
					goto walk
			`	You turn the corner to find the five teenagers gathered there. They seem surprised by your appearance, but Howard hastily addresses you. "Captain! We're so grateful you brought us on board!" The other teens murmur in agreement.`
			`	"Anyway, we don't mean to bother you," he stammers and walks off, the rest following suit.`
			label walk

> 			`	You walk back to your quarters, unsure about what just happened.`

Reviewer 4: I think there should be some option to ask them about what you just heard here.

	on enter
		dialog
			`You hear muffled noises and some awful, out of tone 'singing'. Howard and his friends seem to be having a party.`
	on enter
		dialog
			`This time the partying noises seem even louder than before. You start to wonder how they got that many drinks on board, since they barely had any baggage.`
			`	Later that night, you wake up to a strange noise near your cabin, but attribute it to the party and fall back asleep.`
	on enter
		conversation
			`Today the party seems to have spilled out of their compartments; in fact, you can see Howard and his friends approaching you in the corridor with tottering steps. They stop right in front of where you are standing, in the door of the cockpit.`
			`	"'Ello, capt'n," Howard says. "I wanted to fly a ship since I was a kid. Fancy showin' us how it's done?" They all appear drunk, with one of them leaning on a nearby bulkhead to keep upright.`
			choice
				`	"Sure, come in."`
					goto come
				`	"Sorry, I can't let you into the cockpit."`

> 			`	"Well, that's too bad then. Larry, would you show the kind captain what you got?" Larry is apparently the name of the guy leaning on the wall. He puts a hand behind his back, and pulls out a -`

Reviewer 4: ```suggestion
			`	"Well, that's too bad then." He looks towards his friend leaning on the wall. "Larry, would you show the kind captain what you got?" He puts a hand behind his back, and pulls out a -`
```
